R-packages/evalcast/vignettes/intro_vignette.Rmd

```diff
@@ -0,0 +1,132 @@
+
+Now that we've made these predictions, we'd like to know how well they perform.  By default, `evalcast` uses the following three performance measures, but these can be easily substituted:
+
+```{r}
+err_measures <- list(wis = weighted_interval_score,
+                     ae = absolute_error,
+                     coverage_80 = interval_coverage(alpha = 0.2))
+```
+
+We may now create scorecards for each forecaster's predictions or for a filtered subset. 
+
+```{r}
+scorecards_yyg <- evaluate_predictions(filter_predictions(predictions_cards_yyg, ahead = 3), 
+                                   err_measures, 
+                                   backfill_buffer = 10)
+pcards_cmu_death3 <- filter_predictions(predictions_cards_cmu, 
+                                        ahead = 3,
+                                        response_data_source = "jhu-csse",
+                                        response_signal = "deaths_incidence_num")
+scorecards_cmu <- evaluate_predictions(pcards_cmu_death3, 
+                                   err_measures, 
+                                   backfill_buffer = 10)
+scorecards <- evaluate_predictions(filter_predictions(predictions_cards, ahead = 3), 
+                                   err_measures, 
+                                   backfill_buffer = 10)
+```
+
+Let's look at the `baseline_forecaster`'s scorecard for 3-epiweek ahead forecasts.
+
+```{r}
+scorecards[[1]]
+```
+
+**Technical note:** What does `backfill_buffer = 10` do?  When we evaluate a forecaster in backtesting we are assuming that we know what actually occurred.  However, in light of backfill, we may not trust the data for some period of time.  The argument `backfill_buffer` allows us to specify how many days until we believe that the data has "settled down" and is unlikely to be updated further.  The choice of this argument will depend on the particular signal you are forecasting.
+
+### Plots of performance measures
+
+`evalcast` provides a number of tools to visually assess a forecaster (including comparisons with other forecasters):
+
+```{r}
+scorecards_list <- c(scorecards_yyg,scorecards_cmu,scorecards)
+plot_measure(scorecards_list, "wis")
+plot_measure(scorecards_list, "ae", type = "dotplot") + scale_x_log10()
+plot_calibration(scorecards_cmu[[1]], type = "wedge")
+plot_calibration(scorecards_cmu[[1]], type = "traditional")
+plot_coverage(list(scorecards_yyg[[1]],
+                  scorecards_cmu[[1]]))
+plot_width(scorecards_cmu[[1]])
+```
+
+
+## Cumulative forecasting
+
+While the above describes incident forecasting, the same `evalcast` functions can also be used for cumulative forecasting.
+
+- For example, for `k`-day-ahead cumulative forecasting, choose a cumulative signal from `covidcast` (e.g., `deaths_cumulative_num`), set `incidence_period = "day"` and `ahead = k`.
+
+- For example, for `k`-week-ahead cumulative epiweek forecasting, do the same as above but with `ahead = 7 * k`.
```

Comment:
It'd be good if we actually carry out examples showing how this would work. (@dajmcdon, this is what I was referring to [here](https://github.com/cmu-delphi/covidcast/issues/351#issuecomment-737514603).)